scripts/collect_stream.py

```
@@ -0,0 +1,538 @@
+        )
+
+        # --- Step 2: sanitize ---
+        log("Step 2: sanitize")
+        replace = args.replace_first and i == 0
+        run_sanitize(dump_dir, args.def_name, trace_dir, args.workloads_per_batch, replace)
+
+        # Verify we got something
+        jsonl_path = trace_dir / "workloads" / op_type / f"{args.def_name}.jsonl"
+        n_total = 0
+        if jsonl_path.exists():
+            n_total = sum(1 for l in jsonl_path.read_text().splitlines() if l.strip())
```

Ruff (0.15.9)

[error] Ambiguous variable name: `l`

(E741)